sycl/test-e2e/Matrix/element_wise_all_ops_int8_packed_impl.hpp

```diff
@@ -53,35 +57,35 @@ void matrix_verify_op(big_matrix<T, M, N> &B, const R ref, OP op) {
            ext::intel::experimental::matrix::joint_matrix_store(
                sg, sub_b,
                accB.template get_multi_ptr<access::decorated::no>() +
-                   (sg_startx * TileM) * N * 4 +
-                   sg_starty / sg_size * TileN * 4,
-               N * 4);
+                   (sg_startx * TileRows / VNNI) * Cols * VNNI +
+                   sg_starty / sg_size * TileCols * VNNI,
+               Cols * VNNI);
          }); // parallel for
    }).wait();
-  assert_ops_ref<T, M, N, R>(bufB.get_host_access(read_only), ref);
+  assert_ops_ref<T, Rows, Cols, R>(bufB.get_host_access(read_only), ref);
 }
 
-static constexpr size_t MATRIX_M = TM * 2;
-static constexpr size_t MATRIX_N = TN * 2;
-int8_t B[MATRIX_M][MATRIX_N];
+template <typename Ta, typename TResult, size_t TK, size_t TN, size_t VNNI>
```

Comment:
nit. I guess "Ta" was used in other tests, because matrix A was tested. In this case matrix B is tested, so Tb makes more sense. But I think the best would be to make it something like Ts - like T source.
```suggestion
template <typename Ts, typename TResult, size_t TK, size_t TN, size_t VNNI>
```